src/test/java/spoon/test/filters/FilterTest.java

```diff
@@ -487,10 +491,11 @@ public void testQueryStepScannWithConsumer() throws Exception {
 		CtQuery<CtClass<?>> l_qv = launcher.getFactory().getModel().getRootPackage().filterChildren(new TypeFilter<>(CtClass.class));
 		
 		assertEquals(0, context.counter);
-		l_qv.forEach(cls->{
+		l_qv.map(cls->{
 			assertTrue(cls instanceof CtClass);
 			context.counter++;
-		});
+			return true;
```

Comment:
interesting idea to use CtFunction as Consumer. There is less classes, but it is not nice. Java 8 has Consumer too and java developers will understand that, so I do not see reason to avoid it.